### PR TITLE
Dev ar 177

### PR DESCRIPTION
**Pull request by Garreth-BetBright**

### CODE REVIEW

File: mamba/web/routing.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def dispatch(self, controller, request):
                     {'content-type': 'text/plain'}
                 ))
         except TypeError as error:
-            log.msg(error, logLevel=logging.WARN)
+            log.info(error, logLevel=logging.WARN)
```
Comment (geerk) [inline mamba/web/routing.py]: logLevel will be ignored if we use syslog

**Comment by geerk** [comment]
Also starting logs are not prefixed with severity and go to standard log file (I think they should go to debug log)

**Comment by Garreth-BetBright** [comment]
Some of the logging is directly from Twisted, this includes the startup log messages, some of the factories log messages and HTTP requests. We'll leave them for now, as we may need to fork Twisted in order to reduce these logs, also the twistd binary gives very few options for overwriting the logging config.

File: mamba/application/app.py [hunk 2]
```diff
@@ -15,11 +15,13 @@
```
Comment (Garreth-BetBright) [inline mamba/application/app.py]: Done.

File: mamba/utils/log.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    args_list[0] = "[%s] %s" % (severity, args_list[0])
+    return args_list
+
+def _prepend_err_severity(severity, args):
+    args_list = list(args)
+    if len(args_list) == 2:
+        args_list[1] = "[%s] %s" % (severity, args_list[1])
+    elif isinstance(args_list[0], (str, unicode)):
+        args_list[0] = "[%s] %s" % (severity, args_list[0])
+
```
Comment (Garreth-BetBright) [inline mamba/utils/log.py]: No, we could of course update the error/failure object directly but I don't think its a particularly good practice.

**Pull request merged**